backend/docker-compose.yaml

```diff
@@ -23,10 +23,21 @@ services:
     ports:
       - 80:80
     environment:
-      POSTGRES_HOSTNAME: ${DATABASE}
+      POSTGRES_HOSTNAME: postgres
     restart: unless-stopped
+    command: npm run serve
     depends_on: 
-      - ${DATABASE}
+      - postgres
+  test:
+    image: rafflebay/test:latest
+    ports:
+      - 31337:31337
```

Comment:
This might need to change to 80, not sure. We need the main one to be 80 because that's the port for HTTP requests from the frontend

Reply:
Fixed this